ribs.py

```diff
@@ -123,18 +141,30 @@ def main():
 
     frame_clock = pg.time.Clock()
 
+    time = 0
+    delta_time = 1 / 60
```

Comment:
Might be best to use the same constant here as on line 150.

i.e. `framerate = 60` `delta_time = 1/framerate` `frame_clock.tick(framerate)`